### PR TITLE
[ARTS-95] Trial Logo on Welcome Page

### PR DESCRIPTION
**Pull request by kedarl-rdb**
## Description
Add Trial Logo on Welcome Page

### Changes
- ARTS-95 - Trial Logo on Welcome Page

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
 <body class="mat-typography">
+  <img src="assets/branding/img/ndph-logo.png" />
```
Comment (alexbaxter) [inline src/index.html:13]: Should the name of the logo asset be a configuration option? We might want to change it per trial. But we could have a common fixed name (e.g. 'trial-logo.png'), or derive from the trial deployment name (e.g. 'mts451.png').

File: src/environments/environment.prod.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const environment = {
   production: true,
   issuer: '',
   clientId: '',
+  trialName: ''
```
Comment (alexbaxter) [inline src/environments/environment.prod.ts:9]: You don't need to add this to the production environment.ts - the name is set as an environment variable during deployment.

**Review by alexbaxter — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (eladiw) [reply src/index.html:13]: I would go with 'logo.png'
we can always change in the future

Comment (kedarl-rdb) [reply src/index.html:13]: Changed to logo.png as suggested by Elad

File: src/environments/environment.prod.ts (hunk 2 above)
Comment (kedarl-rdb) [reply src/environments/environment.prod.ts:9]: ng build --prod
The build fails expecting trialName variable
hence as a fix, added the same

**Review by alexbaxter — APPROVED** [review]
OK to merge

File: src/index.html (hunk 1 above)
Comment (eladiw) [inline src/index.html:13]: should be logo.png

Comment (kedarl-rdb) [reply src/index.html:13]: Fixed

**Review by eladiw — APPROVED** [review]

**Pull request merged**